lib/bigquery/dataset.js

```diff
@@ -56,6 +56,9 @@ function Dataset(bigQuery, id) {
 /**
  * Create a table given a tableId or configuration object.
  *
+ * See [Tables: insert](https://cloud.google.com/bigquery/docs/reference/v2/tables/insert)
```

Comment:
Yeah, I'm looking for an option to ignore a comment blocks line length or something similar. If there isn't anything like that I'll just generate short links for all of them